api/tests/core/bookings/test_api.py

```diff
@@ -535,6 +536,69 @@ def test_book_manual_offer(self):
             assert booking.token
             assert len(booking.externalBookings) == 0
 
+        @override_features(ENABLE_CDS_IMPLEMENTATION=True, DISABLE_CDS_EXTERNAL_BOOKINGS=True)
+        def test_should_raise_error_when_cds_external_bookings_are_disabled(self):
+            beneficiary = users_factories.BeneficiaryGrant18Factory()
+            cds_provider = get_provider_by_local_class("CDSStocks")
+            venue_provider = providers_factories.VenueProviderFactory(provider=cds_provider)
+            cinema_provider_pivot = providers_factories.CinemaProviderPivotFactory(venue=venue_provider.venue)
+            offer_solo = offers_factories.EventOfferFactory(
+                name="Séance ciné solo",
+                venue=venue_provider.venue,
+                subcategoryId=subcategories.SEANCE_CINE.id,
+                lastProviderId=cinema_provider_pivot.provider.id,
+            )
+            stock_solo = offers_factories.EventStockFactory(offer=offer_solo, idAtProviders="1111%4444#111/datetime")
+
+            # When
+            with pytest.raises(feature.DisabledFeatureError) as exc:
+                api.book_offer(beneficiary=beneficiary, stock_id=stock_solo.id, quantity=1)
+
+            assert Booking.query.count() == 0
+            assert str(exc.value) == "DISABLE_CDS_EXTERNAL_BOOKINGS is active"
+
+        @override_features(ENABLE_BOOST_API_INTEGRATION=True, DISABLE_BOOST_EXTERNAL_BOOKINGS=True)
```

Comment:
```suggestion
        @override_features(DISABLE_BOOST_EXTERNAL_BOOKINGS=True)
```
`ENABLE_BOOST_API_INTEGRATION` est activé par défaut